docs/core/deploying/publish.md

```diff
@@ -0,0 +1,173 @@
+
+### Why create a framework-dependent deployment?
+
+Deploying an FDD has a number of advantages:
+
+- You don't have to define the target operating systems that your .NET Core app will run on in advance. Because .NET Core uses a common PE file format for executables and libraries independent of the operating system, .NET Core can execute your app regardless of the underlying operating system. For more information on the PE file format, see .NET Assembly File Format.
+- The size of your deployment package is small. You only deploy your app and its dependencies, not .NET Core itself.
+- Unless overridden, FDDs will use the latest serviced runtime installed on the target system. This allows your application to use the latest patched version of the .NET Core runtime.
+- Multiple apps use the same .NET Core installation, which reduces both disk space and memory usage on host systems.
+
+There are also a few disadvantages:
+
+- Your app can run only if the version of .NET Core your app targets, [or a later version](../versions/selection.md#framework-dependent-apps-roll-forward), is already installed on the host system.
+
+- It's possible for the .NET Core runtime and libraries to change without your knowledge in future releases and in rare cases may change the behavior of your app.
+
+Framework-dependent deployment is available for Windows Client, Windows Server, Mac, and Linux platforms through native installers – MSI on Windows Client and Server, PKG on Mac and various native package managers for Linux. 
+
+To create a self-contained deployment, you would use the following command line:
+
+```dotnetcli
+dotnet publish -c Release --self-contained false
+```
+
+
+## Self-contained deployments (SCD)
+
+For a self-contained deployment, you deploy your app and any required third-party dependencies along with the version of .NET Core that you used to build the app. Creating an SCD doesn't include the [native dependencies of .NET Core](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md) on various platforms, so these dependencies must be present before the app runs. For more information on version binding at runtime, see the article on [version binding in .NET Core](../versions/selection.md).
+
+Starting with NET Core 2.1 SDK (version 2.1.300), .NET Core supports *patch version roll-forward*. When you create a self-contained deployment, .NET Core tools automatically include the latest serviced runtime of the .NET Core version that your application targets. (The latest serviced runtime includes security patches and other bug fixes.) The serviced runtime does not have to be present on your build system; it is downloaded automatically from NuGet.org. For more information, including instructions on how to opt out of patch version roll forward, see [Self-contained deployment runtime roll forward](runtime-patch-selection.md).
```

Comment:
Just in case a 2.0 customer (though out of support) lands on teh deployment guide we want to have a way to ensure they dont expect SCD to work (because it isnt supported before 2.1.300). Also I dont think having this phrase takes anything away, so I'd rather leave it in for clarity. If we rev this content in a year or so we could revisit this.